app/routers/event.py

```diff
@@ -251,3 +271,20 @@ def delete_event(event_id: int,
         # if the deletion is successful
     return RedirectResponse(
         url="/calendar", status_code=status.HTTP_200_OK)
+
+
+def get_location_coordinates(
+        address: str,
+        timeout: float = LOCATION_TIMEOUT
+        ) -> Optional[Tuple[float, float, str]]:
+    """Return location coordinates and accurate
+    address of the specified location."""
+    geolocator = Nominatim(user_agent="calendar", timeout=timeout)
+    try:
+        location = geolocator.geocode(address)
+        if location is not None:
+            acc_address = location.raw["display_name"]
+            return location.latitude, location.longitude, acc_address
+    except (GeocoderTimedOut, GeocoderUnavailable) as e:
+        logger.exception(str(e))
+    return None
```

Comment:
You can return (None, None, location), and then you could remove `if location_details is not None:` in line 64

Reply:
Good idea 👍